qemu/tests/balloon_service.py

```diff
@@ -11,24 +11,27 @@
 from aexpect import ShellCmdError
 from virttest.utils_test.qemu import MemoryBaseTest
 from qemu.tests.balloon_check import BallooningTestWin
+from qemu.tests.balloon_check import BallooningTestLinux
 
 
 @error_context.context_aware
 def run(test, params, env):
     """
-    Balloon service test for windows guest.
-    1) boot a windows guest with balloon device.
-    2) enable and check driver verifier in guest.
-    3) install balloon service in guest.
+    Balloon service test, i.e. guest-stats-polling-interval test.
+    1) boot a guest with balloon device.
+    2) enable and check driver verifier in guest(only for windows guest).
+    3) install balloon service in guest(only for windows guest).
     4) enable polling in qmp.
-    5) evict and enlarge balloon.
-    6) get polling value in qmp.
-    7) uninstall balloon service and clear driver verifier.
+    5) evict / enlarge balloon.
+    6) get polling value in qmp, then do memory check if necessary.
+    7) repeat steps 5) and 6) for multi times
+    8) uninstall balloon service and clear driver verifier(only for
+       windows guest).
     """
 
     def get_disk_vol(session):
         """
-        Get virtio-win disk volume letter.
+        Get virtio-win disk volume letter for windows guest.
```

Comment:
virtio-win is only for windows guest.
anywey ok to add "for windows guest" or not